modules/registry-configuring-storage-azure-gov-cloud.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+[id="registry-configuring-storage-azure-gov-cloud_{context}"]
+= Configuring registry storage for Azure Government Cloud
```

Comment:
Totally a judgment call, but unlike AWS GovCloud, I've usually only seen Azure's gov region product called [Azure Government](https://azure.microsoft.com/en-us/global-infrastructure/government/get-started/) (or MAG). With that being said, the *Cloud* part could be dropped.